static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -491,6 +533,20 @@ function WidgetViewerModal(props: Props) {
       </StyledHeader>
       <Body>{renderWidgetViewer()}</Body>
       <StyledFooter>
+        <TotalResultsContainer>
+          {totalResults &&
+            (widget.widgetType === WidgetType.ISSUE ? (
+              <span>
+                <b>{t('Total Issues: ')}</b>
+                {totalResults === '1000' ? '1000+' : totalResults}
```

Comment:
you could also have written as following

```suggestion
               {tct('[description:Total Issues:] [total].', {
                  description: <strong />,
                  total: totalResults === '1000' ? '1000+' : totalResults,
                })}
```

but it looks good to me